.travis.yml

```diff
@@ -25,10 +23,12 @@ before_install:
 
 install:
   - |
+    export installDir=$(pwd);
```

Comment:
Wofür brauchen wir das solange wir immer in dem Verzeichnis bleiben?

Reply:
falls wir z.B. mal BUILD und FORMAT gleichzeitig machen wollen. Der eigentliche Fehler war ja, dass ein cd leptonica... fehlschlug, und danach einfach das cd .. dennoch gemacht wurde. d.h. es wäre sicherer, vor jedem Schritt ein cd $installDir zu machen.